sparkmagic/sparkmagic/livyclientlib/sparkcommand.py

```diff
@@ -0,0 +1,120 @@
+from sparkmagic.utils.utils import coerce_pandas_df_to_numeric_datetime, records_to_dataframe
+import sparkmagic.utils.configuration as conf
+import sparkmagic.utils.constants as constants
+from sparkmagic.utils.sparkevents import SparkEvents
+from sparkmagic.livyclientlib.command import Command
+from sparkmagic.livyclientlib.exceptions import DataFrameParseException, BadUserDataException
+
+import ast
+
+class SparkStoreCommand(Command):
```

Comment:
Structurally this class looks quite a lot like the `SqlQuery` class.  Can/should this be refactored?